src/api/spec/browser_helper.rb

```diff
@@ -7,3 +7,7 @@
 
 # helper methods for authentification in features tests
 require 'support/features/features_authentification'
+
+# Shared examples. Per recommendation of RSpec,
+# https://www.relishapp.com/rspec/rspec-core/v/2-12/docs/example-groups/shared-examples
+Dir["./spec/support/shared_examples/features/*.rb"].each { |example| require example }
```

Comment:
Added documentation for it. I tend to think that we should move all shared examples into under spec/support/shared_examples